app/src/main/java/com/sdp13epfl2021/projmag/MainActivity.kt

```diff
@@ -33,7 +33,7 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        UserTypeChoice.isProfessor = getSharedPreferences(UserTypeChoice.savedTypeChoice, Context.MODE_PRIVATE).getBoolean(UserTypeChoice.isUserProf, true)
+        UserTypeChoice.isProfessor = true
```

Comment:
This seems to be used for testing. change should be revert.

Reply:
Done